tarball-ingester.tf

```diff
@@ -511,3 +520,76 @@ resource "aws_iam_policy" "minio_credentials_secretsmanager" {
   description = "Allow reading of MinIO Access and Secret Keys"
   policy      = data.aws_iam_policy_document.minio_credentials_secretsmanager.json
 }
+
+resource "aws_lb" "tarball_ingester" {
+  name               = "tarball-ingester"
+  internal           = true
+  load_balancer_type = "network"
+  subnets            = data.terraform_remote_state.ingest.outputs.ingestion_subnets.id
+  tags               = local.common_tags
+
+  access_logs {
+    bucket  = data.terraform_remote_state.security-tools.outputs.logstore_bucket.id
+    prefix  = "ELBLogs/tarball-ingester"
+    enabled = true
+  }
+}
+
+resource "aws_lb_target_group" "tarball_ingester" {
+  name_prefix       = "ti-"
+  port              = 9000
+  protocol          = "TCP"
+  target_type       = "instance"
+  vpc_id            = data.terraform_remote_state.ingest.outputs.vpc.vpc.vpc.id
+  proxy_protocol_v2 = true
```

Comment:
Don't need that here, thankfully. It's a bit of a sec vuln having it enabled (it's only needed for Squid)